src/webui/www/private/scripts/client.js

```diff
@@ -175,6 +175,15 @@ let setStatusFilter = () => {};
 let toggleFilterDisplay = () => {};
 
 window.addEventListener("DOMContentLoaded", (event) => {
+    document.id = el => {
+        switch (typeOf(el)) {
+        case "string":
+            return document.getElementById(el);
+        case "element":
+            return el;
+        }
+    };
+
```

Comment:
Please move it after `window.qBittorrent.LocalPreferences.upgrade();`

Also please add brief comments about it (it isn't apparent about what it is doing).